### DISCUSSION TITLE
Indexing when create | recreate  collection

### DISCUSSION THREAD

**Discussion by AbdallahTarabish** [jenssegers/laravel-mongodb#2812] [Q&A] [answered]
when  delete a collection and recreate it, the indexes associated with that collection are not automatically recreated ,this is because index creation is not a default behavior during collection creation,
what's the solution ?

**Comment by GromNaN** [comment, marked as answer]
Indexes are part of the schema management feature. You can use Laravel Migrations to manage your indexes.
https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/schema-builder/#manage-indexes
